charts/dvpe-netfilter/README.md

```diff
@@ -0,0 +1,37 @@
+# dvpe-netfilter
+
+![Version: 0.1](https://img.shields.io/badge/Version-0.1-informational?style=flat-square)
+
+Helm chart for set netfilter rules on Kubelet node
+
+## Installation
+Installs dvpe-helm netfilter injector on an existing Kubernetes cluster.
+
+**Note**: The netfilter injector needs to run as root with NET_ADMIN capabilties and use hostNetwork in namespace kube-system.
```

Comment:
I would add this additional sentence: 

`When Pod Security Policies are used in your cluster, then they need to reflect those requirements.`